### ISSUE TITLE
No config file and very little content

### ISSUE THREAD

**Issue by pastasan** [Luke100000/minecraft-comes-alive#36]
Hello everyone
I downloaded this on curseforge to both my client and server ( https://www.curseforge.com/minecraft/mc-mods/minecraft-comes-alive-testing)
For some reason only the mca_help + mca_editor commands work, and there is no config file.
What might be wrong?

**Comment by Luke100000** [comment]
Hard to say. Can you define "not working", and maybe you could append the log.

**Comment by pastasan** [comment]
Hello there Luke.
The log doesn't mention the mod at all. The only "Notice" I get of it is when logging in (I can access the mca editor and mca help) the latter shows 6 available commands. And that's it. Otherwise the mod seems dormant and there is no option to configure it (no config file). I ran the 1.16.5. server jar and the latest jar version of the minecraft comes alive reborn + fabric that I downloaded from curseforge. Same thing happened in my client version -- no config file.
